src/core/heating_cooling_loads.jl

```diff
@@ -94,7 +95,8 @@ struct DomesticHotWaterLoad
         end
 
         new(
-            loads_kw
+            loads_kw,
+            (sum(loads_kw)/time_steps_per_hour)/KWH_PER_MMBTU
```

Comment:
No not really because loads_kw isn't an input, it is calculated based on fuel_loads_mmbtu_per_hour or reference profile, annual_mmbtu, and monthly_mmbtu. If the user provides both, then fuel_loads_mmbtu_per_hour is used, so we could add a warning about that. But if the user provides doe_reference_name etc, then that is used to calculated loads_kw, so this should be the same value as provided.